app/bot/when.go

```diff
@@ -0,0 +1,134 @@
+package bot
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+)
+
+// When bot is answer on question "when the stream is started".
+type When struct {
+	now     func() time.Time
```

Comment:
would be helpful to add a comment "for testing usage" or smth like this

Reply:
actually, I think that is over engineering and should be deleted completely.

Reply:
this deletion will simplify `NewWhen`

Reply:
fixed